aws-iam-role-ci-user-manager/main.tf

```diff
@@ -47,20 +53,6 @@ data "aws_iam_policy_document" "ci-manager" {
 
     resources = ["*"] // list all users to be able to use the console
   }
-
-  statement {
-    sid = "ExternalPartyAssume"
-    principals {
-      type        = "AWS"
-      identifiers = ["arn:aws:iam::${var.caller_account_id}:root"]
-    }
-
-    condition {
-      test     = "StringEquals"
-      variable = "sts:ExternalId"
-      values   = [var.sts_external_id]
-    }
-  }
 }
 
 resource "aws_iam_role_policy" "ci-manager" {
```

Comment:
⚠️ **[tflint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Missing version constraint for provider "aws" in "required_providers"